example/sample.cc

```diff
@@ -42,8 +42,9 @@ int main() {
     std::string context = req.get_header_value(kPropagationHeader.data());
 
     TracingContextPtr tracing_context;
+
     if (!context.empty()) {
-      // 2. Create tracing context with propagated information.
+      // 2. Create segment context with propagated information.
```

Comment:
I think this is a misoperation? I think `create tracing context` is correct from word reading.